backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Services/ResourceAdministrationPoint.cs

```diff
@@ -79,30 +75,30 @@ public async Task<List<ServiceResource>> GetResources(string scopes)
         }
 
         /// <inheritdoc />
-        public async Task<List<ServiceResource>> GetResources(List<Tuple<string, string>> resourceIds)
+        public async Task<List<ServiceResource>> GetResources(List<string> resourceIds)
         {
             List<ServiceResource> filteredResources = new List<ServiceResource>();
 
             try
             {
-                foreach (Tuple<string, string> id in resourceIds)
+                foreach (string id in resourceIds)
                 {
                     ServiceResource resource = null;
 
-                    resource = await GetResource(id.Item1);
+                    resource = await GetResource(id);
 
                     if (resource == null)
                     {
                         ServiceResource unavailableResource = new ServiceResource
                         {
-                            Identifier = id.Item1,
+                            Identifier = id,
                             Title = new Dictionary<string, string>
                         {
                             { "en", "Not Available" },
                             { "nb-no", "ikke tilgjengelig" },
                             { "nn-no", "ikkje tilgjengelig" }
                         },
-                            ResourceType = Enum.TryParse<ResourceType>(id.Item2, out ResourceType resourceType) ? resourceType : ResourceType.Default
+                            ResourceType = ResourceType.Default
```

Comment:
Why is resourcetype set to <<Default>> here?

Reply:
The backend does not include resourcetype anymore, and this is setting default resource data when it's not found in the resource registry, so it just makes sense (to me) to have it set to default.
I also see now that it's also using the "nb-no" and "nn-no" language codes which won't work with the user language code